### PR TITLE
Updated the A40 test case to use a pipeline instead of a manual install

### PR DESCRIPTION
**Pull request by martinhesko**
# Issue link
[MGDAPI-2336](https://issues.redhat.com/browse/MGDAPI-2336)

# What
Updated the A40 testcase to automate the deployment of OSD Trial cluster and RHOAM install through an updated install-master pipeline

Not to be merged until [the update to master-install](https://gitlab.cee.redhat.com/integreatly-qe/ci-cd/-/merge_requests/572) pipeline is merged

# Verification steps
N/A

### CODE REVIEW

File: test-cases/tests/installation/a40-rhoam-on-osd-trial.md [hunk 1]
```diff
@@ -18,42 +16,31 @@ Verify RHOAM installation on OSD Trial works as expected.
 
 ## Steps
 
-1. Provision an OSD Trial Cluster
+1. Provision an OSD Trial Cluster through Jenkins using the `managed-api-install-master` pipeline
```
Comment (trepel) [inline test-cases/tests/installation/a40-rhoam-on-osd-trial.md:19]: I would add a link to the pipeline here. Syntax:  `[text of the link](url)`

Comment (trepel) [reply test-cases/tests/installation/a40-rhoam-on-osd-trial.md:19]: lgtm otherwise

Comment (martinhesko) [reply test-cases/tests/installation/a40-rhoam-on-osd-trial.md]: Should be good now

**Pull request merged**